visdag_tests/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Blank line.

Reply:
Okay, this is weird. I didn't touch any of the files in visdag_tests, so it looks like dagster is creating these __init__.py files with an empty line. Not sure why. Anyway, that's fixed.